processing/src/main/java/org/apache/druid/query/metadata/metadata/AllColumnIncluderator.java

```diff
@@ -32,7 +32,7 @@ public boolean include(String columnName)
   @Override
   public byte[] getCacheKey()
   {
-    return ALL_CACHE_PREFIX;
+    return new byte[]{0x1};
```

Comment:
It would be better to store these byte arrays as static fields in the subclass of `ColumnIncluderator`.

Reply:
I agree, let me update the PR